festim/mesh/mesh_1d.py

```diff
@@ -23,6 +23,14 @@ def __init__(self, vertices, **kwargs) -> None:
         mesh = self.generate_mesh()
         super().__init__(mesh=mesh, **kwargs)
 
+    @property
+    def vertices(self):
+        return self._vertices
+
+    @vertices.setter
+    def vertices(self, value):
+        self._vertices = np.sort(np.unique(np.array(value, dtype=np.float64)))
```

Comment:
This looks a bit redundant as np.unique returns a `np.ndarray`
Why enforcing `dtype` `np.float64` ?
```suggestion
        self._vertices = np.sort(np.unique(value))
```

Reply:
Using this allows users to have either a list of floats or ints, as if we remove these and a list of ints is given, a RuntimeError is raised

Reply:
I don't seem to have an issue. See below

```
>>> import numpy as np
>>> a = [1, 2, 3]
>>> np.unique(a)
array([1, 2, 3])
>>> np.sort(np.unique(a))
array([1, 2, 3])
```

Reply:
It seems that putting only np.sort(np.unique(value)) is not enough and retrun an error in the tests :
=========================== short test summary info ============================
FAILED test/test_mesh.py::test_mesh_vertices_from_list - RuntimeError: Unsupported mesh dtype: int64

Reply:
In that case I would do something like:

```suggestion
        self._vertices = np.sort(np.unique(value)).astype(float)
```